uisdk/src/main/java/com/karhoo/uisdk/util/TimeUtil.kt

```diff
@@ -0,0 +1,26 @@
+package com.karhoo.uisdk.util
+
+import android.content.Context
+import com.karhoo.uisdk.R
+
+object TimeUtil {
+    fun roundMinutesInHours(minutes: Int): Int {
+        return minutes / HOUR_IN_MINUTES
+    }
+
+    fun getLeftOverMinutesFromHours(minutes: Int): Int {
+        return minutes % HOUR_IN_MINUTES
+    }
+
+    fun getHourAndMinutesFormattedText(context: Context, minutes: Int, hours: Int): String {
```

Comment:
Could this function just take the minutes as params. It will transform the minutes into hours + minutes itself. No need to expose the 2 functions above. I don't see how they will be used in other places other than creating this duration string.

Reply:
Don't think it's mandatory, we could have any type of time units (millis, seconds, minutes, hours, etc) so pre-parsing the time units will still be needed before calling the function whether it takes minutes + hours or just minutes. Also, if we'd like to only have one parameter, we should have a duration param that's milliseconds and not minutes/hour. 
Guess, it's just a matter of preference